apps/pipelines/models.py

```diff
@@ -277,7 +278,10 @@ def invoke(
                 ai_message = self._save_message_to_history(
                     session, output["messages"][-1], ChatMessageType.AI, metadata=output_metadata
                 )
-                output["ai_message_id"] = ai_message.id
+                pipeline_output = ai_message
+            else:
+                # Return the pipeline state. This is currently for logging purposes only.
+                pipeline_output = output
```

Comment:
Should we rather instantiate `ChatMessage` with the state as the content here?

Reply:
Yes, I think we should do that

Reply:
https://github.com/dimagi/open-chat-studio/pull/1414/commits/d8e6e512ea7278cdbdb1b1de383b892c0cb96e44